pkg/trie/triedb/proof/proof_test.go

```diff
@@ -1,22 +1,19 @@
-// Copyright 2024 ChainSafe Systems (ON)
-// SPDX-License-Identifier: LGPL-3.0-only
-
```

Comment:
nit/need license

Reply:
Good catch, don't know why I deleted it 👀